lib/network/modules/View.js

```diff
@@ -138,6 +138,13 @@ class View {
       options = {};
       return;
     }
+
+    // Coerce and verify that the scale is valid.
+    options.scale = +options.scale
+    if (!(options.scale > 0)) {
+      throw new TypeError('Scale has to be a number bigger than zero.')
```

Comment:
How about? `'scale' has to be a number bigger than zero.`

Reply:
Probably a bit more explicit. Will do.